app/controllers/cv_search_controller.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+class CvSearchController < ApplicationController
+
+  def index
+    @cv_search_results = query.nil? || query.length < 3 ? [] : search_results
```

Comment:
`query.nil? || query.length < 3` I would extrapolate this to a seperate method named something like `should_search` and then use that for the ternary because otherwise the ternary is not readable anymore